.eslintignore

```diff
@@ -1 +1,2 @@
-/coverage
\ No newline at end of file
+/coverage
+/src/utils/sendMessages.js
```

Comment:
Do you have a preference on using an `eslint-disable` comment as opposed to disabling linting for the entire file?